### ISSUE TITLE
Suggestions for the current /mission page

### ISSUE THREAD

**Issue by chromy** [QueerCodeOrg/queercodeberlin.github.io#30]
Split from #29:

> [...]
>
> I'd also consider adding an intersectional statement about how people of diverse gender and sexuality are also of many races, abilities, religions, etc... and that there is no hierarchy of oppression.
> 
> BTW I strongly dislike the sentiment behind statements like "We want to be with people like us" and/or "a place for like-minded people" - I personally think they are anti-diversity statements that suggest there is a shared commonality between queer people and we are all the same on some level. I think this is colonialist thought that stands in the way of true diversity but I understand the thought behind it :)
> 
> In terms of "the identities" I prefer "people of all genders and sexualities" as this doesn't require self-identification (something fraught in itself) personally.
> 
> What do you lot think?
> 
> Kim

@kimadactyl mentions three suggestions with respect to the text on the current mission page (I will attempt to summarise/number):
1. We should add an intersectional statement.
1. We should reconsider the "We want to be with people like us" language currently present.
1. We should consider replacing the big list of identities with someone like “people of all genders and sexualities”.

@kimadactyl does that seem fair and accurate? (Particularly number three which I wasn’t sure about).

**Comment by kimadactyl** [comment]
1 and 2: yes!
3: No not replacing! That was more suggested as a shorthand where needed for the list as a whole as it gets a little cumbersome :) (although might want to add "questioning and unsure" at the end)?

**Comment by dirv** [comment]
This is perhaps a good candidate for adding to our values statement (not yet present, see Issue #36)

**Comment by kimadactyl** [comment]
I don't think so, intersectionality is a black feminist concept about the overlap of oppressions, not interpersonal difference. See for e.g. http://theangryblackwoman.com/2009/08/02/intersectionality/
